docs/protocol/marketplace.md

```diff
@@ -0,0 +1,53 @@
+OpenPAI Marketplace
+===================
+
+OpenPAI marketplace is a service for finding and sharing your PAI job or the job's prerequisite assets.
+It will provide the following features:
+
+1. Protocol
+2. Repository
+3. Marketplace
+
+
+Protocol
+--------
+
+OpenPAI protocol is a set of schema which defines all required or optional aspects of an OpenPAI job. It has five types of components: _JOB_, _DATA_, _SCRIPT_, _DOCKERIMAGE_, and _OUTPUT_. The _JOB_ type may require other types which are prerequisite assets for an AI job.
+
+The protocol for a specific job can be written as a YAML file, which is called protocol file. A protocol file is valid if and only if it meets the schema in protocol.
+
+To implement the protocol, we need to define the schema for job and validate a YAML file for the pre-defined schema.
+
```

Comment:
define or extend? We have a base PAI protocol, right?

Reply:
Right. This part has been implemented and it refers to existing [protocol](https://github.com/Microsoft/pai/blob/pai-proto/docs/pai-protocol.yaml).